index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div id="main">
+        <section id="navigation">
+            <nav id="nav-bar">
+                <ul>
+                    <li><a href="index.html">Youtube</a></li>
+                </ul>
+            </nav>
+        </section>
+            
+        <section id="sidebar">
+            <div>
+                <img src="./img/thumbnail2.jpeg" alt="">
+            </div>
+            <div>
+                <img src="./img/thumbnail3.jpeg" alt="">
+            </div>
+            <div>
+                <img src="./img/thumbnail2.jpeg" alt="">
+            </div>
+            <div>
+                <img src="./img/thumbnail3.jpeg" alt="">
+            </div>
+        </section>
```

Comment:
Please add an alt text to your <img> tags. Using alt text allows you to improve your page SEO that can bring more users to your page.

Reply:
- [ ] Kindly make sure you provide the truthful `reason` on why the changes were requested.
`alt` tag is rather used for enhancing the user experience `UX`. that is, if the image did not load or does not display on the page for different reasons the user see the text which was put in the `alt`
You can check more about `alt` attribute [here](https://www.w3schools.com/tags/att_area_alt.asp#:~:text=Definition%20and%20Usage,user%20uses%20a%20screen%20reader).